Source/JavaScriptCore/inspector/agents/InspectorConsoleAgent.h

```diff
@@ -99,6 +100,7 @@ class JS_EXPORT_PRIVATE InspectorConsoleAgent : public InspectorAgentBase, publi
     HashMap<String, MonotonicTime> m_times;
     bool m_enabled { false };
     bool m_isAddingMessageToFrontend { false };
+    bool m_consoleClearAPIEnabled { false };
```

Comment:
Is there a reason for this flag to be default `false` while the frontend setting is default `true`?

Reply:
eh i generally err on the side of the default value being `false` since it keeps the memory as `0`.  i could rework this to be `m_consoleClearAPIDisabled` but generally across Web Inspector we prefer `*Enabled`

Reply:
The naming is good 👍, no need to change that. 

It's just a bit odd that the backend starts in a default `false` state and expects the frontend to flip it to `true` which is the _real_ default state. Someone reading this later might be confused and question whether there's some subtle behavior being accounted for.

I'd suggest keeping the two in sync for clarity.